libfreerdp/core/nla.c

```diff
@@ -1790,3 +1790,11 @@ DWORD nla_get_error(rdpNla* nla)
 		return ERROR_INTERNAL_ERROR;
 	return nla->errorCode;
 }
+
+UINT32 nla_get_sspi_error(rdpNla* nla)
+{
+	if (!nla || !nla->auth)
```

Comment:
better `WINPR_ASSERT` these, at least `nla` must not be `NULL`

Reply:
fixed